src/rencache.c

```diff
@@ -134,7 +135,7 @@ void rencache_draw_rect(RenRect rect, RenColor color) {
   }
 }
 
-float rencache_draw_text(lua_State *L, RenFont **fonts, const char *text, float x, int y, RenColor color)
+float rencache_draw_text(UNUSED lua_State *L, RenFont **fonts, const char *text, float x, int y, RenColor color)
```

Comment:
Let's just remove the parameter, if it's not used, unless there's a good reason to keep it around.